data/fields/artwork_type.json

```
@@ -12,6 +12,9 @@
             "bust": "Bust",
             "painting": "Painting",
             "mosaic": "Mosaic",
+            "tilework": "Tile work",
+            "stone": "Stone",
+            "street_art": "Street-art",
             "relief": "Relief"
```

Maybe it's better to sort them approximately by usage. Also, let's change spelling to title case and use slightly more specific labels. The tag value for street art is more commonly `streetart` without `_`.

```suggestion
            "relief": "Relief",
            "stone": "Artistically Worked Stone",
            "streetart": "Other Street Art",
            "tilework": "Tile Work"
```

---

>The tag value for street art is more commonly streetart without _.

This is what I actually was hoping to improve as I see `street_art` as a better/more proper value. So I'd prefer to keep it with underscore.

---

> no real consensus in general […] but way broader

That would indicate even more towards not including it in this repository, as long as there is no real common definition for OSM of these types of artworks.

I would tend towards omitting the value from this PR.

---

Ok, let's drop it